pyof/foundation/basic_types.py

```diff
@@ -531,6 +552,7 @@ def pack(self, value=None):
             msg = "{} pack error: {}".format(type(self).__name__, err)
             raise exceptions.PackException(msg)
 
+    # pylint: disable-msg=arguments-differ
```

Comment:
This should not affect the rest of the class. Either write in the same lame as `def` (recommended) or enable it right after the method.

Reply:
@cemsbr `same lame` == `same line` ?